From 63ae7a29d8389f8a6fb2e7cee4fa32f5df191f6e Mon Sep 17 00:00:00 2001
From: Rob Crittenden <rcritten@redhat.com>
Date: Fri, 26 Oct 2012 13:31:22 -0400
Subject: [PATCH] The SECURE_NFS value needs to be lower-case yes on SysV systems.

The sysV rpcgssd init script tests for [ "${SECURE_NFS}" != "yes" ].

This also works as lower case for system so a simple fix.

https://fedorahosted.org/freeipa/ticket/3207
---
 ipa-client/ipa-install/ipa-client-automount |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/ipa-client/ipa-install/ipa-client-automount b/ipa-client/ipa-install/ipa-client-automount
index 7ea69ec12be12f1e0d50abc9ef59a979aa25c7a1..182f9f659dc3bfca198fc254f1a51047b3d9d31f 100755
--- a/ipa-client/ipa-install/ipa-client-automount
+++ b/ipa-client/ipa-install/ipa-client-automount
@@ -304,7 +304,7 @@ def configure_nfs(fstore, statestore):
     Configure secure NFS
     """
     replacevars = {
-        'SECURE_NFS': 'YES',
+        'SECURE_NFS': 'yes',
     }
     ipautil.backup_config_and_replace_variables(fstore,
         NFS_CONF, replacevars=replacevars)
-- 
1.6.0.6

